scripts/generate_site_sidebar/sidebar_template.md.tmpl

```diff
@@ -56,7 +56,8 @@
   - [Contrib](https://catalog.kpt.dev/contrib/ ":target=_self")
 - [Guides](guides/)
   - [The Rationale behind kpt](guides/rationale.md)
-  - [Namespace provisioning example](guides/namespace-provisioning-example.md)
+  - [Namespace provisioning CLI](guides/namespace-provisioning-cli.md)
+  - [Namespace provisioning UI](guides/namespace-provisioning-ui.md)
```

Comment:
I feel like removing the "example" could be confusing here：1. it seems to be the prerequisite to use Porch. 2. it's placed right after "Rationale" and before "Porch", which seems to be the second thing users should be consider (even before installing porch?)